Index: src/physics/MoveEngine.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/physics/MoveEngine.java	(date 1457006839000)
+++ src/physics/MoveEngine.java	(revision )
@@ -127,7 +127,7 @@
         double relY = s.getCenterY() - t.getCenterY();
         // Take the arctan to find the collision angle.
         double collisionAngle = Math.atan2(relY, relX);
-        while (collisionAngle < 0) collisionAngle += 2 * Math.PI;
+        while (collisionAngle > 0) collisionAngle += 2 * Math.PI;
         // Rotate the coordinate systems for each object's velocity to align
         // with the collision angle. We do this by supplying the collision angle
         // to the vector's rotateCoordinates method.
